src/devices/cpu/gigatron/gigatron.h

```diff
@@ -0,0 +1,68 @@
+// license:BSD-3-Clause
+// copyright-holders:<author_name>
+/*****************************************************************************
+ *
+ * Skeleton Device for Gigatron CPU Core
+ *
+ *****************************************************************************/
+
+#ifndef MAME_CPU_GTRON_H
+#define MAME_CPU_GTRON_H
```

Comment:
Please make this match the file path properly - it should be `MAME_CPU_GIGATRON_GIGATRON_H`